### ISSUE TITLE
LiveSplit 1.6.9 Hogs too much CPU usage on Windows 10

### ISSUE THREAD

**Issue by GrowthKasei** [LiveSplit/LiveSplit#508]
Hi.

So I've been using LiveSplit for a while now, primarily on Windows 8.1 Pro where it worked just fine. Recently, however, I upgraded to Windows 10 and ever since that point, LiveSplit has started hogging so much of my CPU usage that I literally cannot stream a PC game and run it at the same time. This is devastating as my main games are typically PC ones.

My processor in general is not the greatest and is quite old and outdated for sure. (I'll list it down below in the specs section.) But there's absolutely no reason as to why it should be acting up this way when it really hasn't been doing it before. A lot of streamers I follow also have Windows 10 and runs LiveSplit just fine - it seems to be just me.

TL;DR - As soon as I run LiveSplit, both with and without splits, its CPU usage goes through the roof, peaking around 30% and using about 44MB of RAM. This only started happening once I upgraded to Windows 10. How can I fix this?

This is while using 1.6.0.
![image](https://cloud.githubusercontent.com/assets/17676367/13549783/9239c0f0-e30d-11e5-9838-2fa719cc12e8.png)

This is while using 1.6.9.
![image 2](https://cloud.githubusercontent.com/assets/17676367/13549806/fce066a2-e30d-11e5-8d4b-4fe65d3c9532.jpg)

Here's what I've done to try and fix it:

1. Set LiveSplit to use one of my four available cores.
2. Have only OBS, my game and LiveSplit running.
3. Tried previous versions of LiveSplit. (1.6.9, 1.6.7, 1.6.0)
4. Completely gotten rid off LiveSplit and re-downloaded it.
5. Made sure .NET Framework is up-to-date, aswell as Java.

Specs:

Intel(R) Core(TM) i3-3220 CPU @ 3.30GHz (4 CPUs), ~3.3GHz
8GB RAM
NVIDIA GeForce GTX 760
Windows 10 Home 64-bit

Thanks in advance! I really have no idea how to fix this and running these games without splits is devastating as I am going for very optimal times, I need to see time gains and stuff like that.

**Comment by wooferzfg** [comment]
are you getting any LiveSplit errors in your Windows Event Viewer?

**Comment by GrowthKasei** [comment]
Yes.

![ss 2016-03-06 at 01 01 30](https://cloud.githubusercontent.com/assets/17676367/13551346/201c6232-e337-11e5-8283-eb71c6442581.png)

![livesplit fel](https://cloud.githubusercontent.com/assets/17676367/13551357/59fd61fe-e337-11e5-960a-efaf5c67ff9a.png)

I'm aware it's all in swedish, I apologize. Everything that says "Fel" basically shows an error. I have no idea what any of these things mean, but all the red marked ones on the images belong to LiveSplit. I made the window to the left bigger so you could see the 'code' - it's fairly similar on every single application error.

If you need me to go more in-depth somehow, just let me know!

**Comment by wooferzfg** [comment]
It's some kind of issue with a device (gamepad, mouse, something else?) that it's trying to pull hotkeys from. Did you get any new/different devices when you switched to Windows 10?

**Comment by GrowthKasei** [comment]
Wow, that is really weird.

There's literally no new device since I upgraded.. however, I started LiveSplit and took my Xbox One controller out of my PC, and the CPU usage went back down to 0.2%. With the timer active, it went up to 4% max. I then put it back in, and it's still got a very low value.

Then I restarted LiveSplit with the controller in, and the CPU went up again, and the same process as before worked to take it back down. I'm honestly clueless. I mean, it works to do this but I just don't know how the controller could interact with LiveSplit?

When I run I use auto splitters, so there's no hotkey assigned to my controller or anything for splitting.

**Comment by wooferzfg** [comment]
For some reason LiveSplit isn't declaring your controller as "broken" and keeps trying to get inuts from it, causing exceptions...

**Comment by fatalis** [comment]
A friend of mine tried to reproduce the issue and couldn't. (Win10 / Xbone wired)

**Comment by GrowthKasei** [comment]
Did some checks and it looks like my Xbone controller lacked drivers, so I updated it and tried again, still the same issue after multiple reboots.

Is my controller just haunted?

**Comment by adrianzech** [comment]
> Is my controller just haunted?

Nope, same issue here.

**Comment by Edacaf** [comment]
My Xbox controller was preventing LiveSplit from even opening.

**Comment by wooferzfg** [comment]
a lot of people are having this issue, so I moved it here https://github.com/LiveSplit/LiveSplit/issues/517
